analysis_options.yaml

```diff
@@ -1,7 +1,19 @@
 analyzer:
   strong-mode:
     implicit-casts: false
-#    implicit-dynamic: false
+    # implicit-dynamic: false
+
+  # Add this to your project `analysis_options.yaml` to enable the built_value
+  # analyzer plugin. You should also add the top level `plugins` clause below.
```

Comment:
Maybe a 5-word summary on what the plugin does here would be good for people copy-pasting it.

Reply:
Done.